products/extensions/b2b-components/employee-management/concept/entities-and-schema.md

```diff
@@ -0,0 +1,47 @@
+# Entities and schema
+
+## Entities
+
+### Company
+
+The company entity represents a company that is registered as a customer in the shop. It is used to gather a company's employees and roles.
+
+### Employee
+
+The employee entity represents a customer working for a specific B2B merchant. It acts on behalf of the associated company to e.g. place orders. A role can be assigned to an employee.
```

Comment:
"The employee entity represents a customer working for a specific B2B merchant" is in my opinion a bit confusing. Maybe: The employee entity represents an employee working for a specific B2B business partner